### DISCUSSION TITLE
VPC v2025.01

### DISCUSSION THREAD

**Discussion by eddie-knight** [finos/common-cloud-controls#587] [Change Management Board (CMB)]
Please use this space to discuss the [release candidate](https://github.com/finos/common-cloud-controls/releases/tag/v2025.01.VPC-rc) for Virtual Private Cloud.

The controls catalog to review may be downloaded here: [CCC.VPC_2025.01.pdf](https://github.com/finos/common-cloud-controls/releases/download/v2025.01.VPC-rc/CCC.VPC_2025.01.pdf)

Please follow this guide for CMB discussions: [CMB Feedback Guide](https://github.com/finos/common-cloud-controls/blob/main/docs/community-guidelines/releases/cmb/feedback-guide.md)

> [!NOTE]
> Remember to use "Add a Comment" to create a new feedback entry, and use "Write a reply" to extend or debate an existing point. Release managers may moderate posts for organizational or conduct purposes.

### Release Manager: Damien Burks (@damienjburks)

**Comment by damienjburks** [comment]
Official Release: https://github.com/finos/common-cloud-controls/releases/tag/v2025.01.VPC

For any additional updates or feedback, please reach out to the ccc-communications+subscribe@lists.finos.org.
